doc/adr/0002-repository-standardization.rst

```diff
@@ -0,0 +1,84 @@
+ADR 2: Repository Standardization
+=======================================
+
+Date: 2022-07-06
+
+Authors
+---------
+
+Lorenzo Calegari <lorenzo.calegari@iohk.io>
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+IOG is undertaking a company-wide effort to restructure and standardize its 
+repositories, favoring mono-repos and enforcing shared GitOps and DevOps 
+processes. Parallel to this, a new CI infrastructure is being developed.
+
+Examples of this are:
+
+* `input-output-hk/cardano-world <https://github.com/input-output-hk/cardano-world>`_
+* `input-output-hk/ci-world <https://github.com/input-output-hk/ci-world>`_
+* `input-output-hk/atala-world <https://github.com/input-output-hk/atala-world>`_
+  
+This initiative appears to be championed by the SRE team who are the creators of 
+`divnix/std <https://github.com/divnix/std>`_. Indeed `std` is at the heart of 
+the standardization dream.
+
+In short, `std` aims to answer the one critical question that pops in the mind 
+of newcomers and veterans alike: 
+
+*What can I do with this repository?*
```

Comment:
Is this something we have a problem with? It's nice, but is it addressing an actual pain point? I thought the driver for this was that you wanted to use it to reduce the duplication, but you don't discuss that in the motivation and you don't say how it will help!

Reply:
Point addressed in `Implications` and `Argument`